index.html

```diff
@@ -309,14 +309,14 @@ <h1 class="greet">
           <ul class="skills">
               <ul class="languages"> 
                 <li><p>Languages</p></li>
-                <li class="arrow"><img src="assets/Unionlangs.png"></li>
+                <li class="arrow"><img src="assets/Unionlangs.png" alt=""></li>
               </ul>
               <div class="langs_list">
               <li class="js_2"> 
-                <img src="assets/jsLogo.png"><p>Javascript</p>
+                <img src="assets/jsLogo.png" alt=""><p>Javascript</p>
```

Comment:
- [ ] Same here

Reply:
Great job on adding the `alt` attribute here 🎉 it really makes your page more accessible. In some cases, it's ok to leave it empty and you did that well in some others but in this case, it's necessary to tell the reader that this is a JavaScript logo, in case it's not rendered. 🤓